zerver/lib/user_groups.py

```diff
@@ -595,11 +615,17 @@ def user_groups_in_realm_serialized(
         for setting_name in NamedUserGroup.GROUP_PERMISSION_SETTINGS:
             group_settings_ids.add(getattr(group, setting_name + "_id"))
 
-    group_ids_to_fetch_members = list(realm_group_ids | group_settings_ids)
+    if realm_setting_group_ids is None:
+        realm_setting_group_ids = set()
+
+    group_ids_to_fetch_members = list(
+        realm_group_ids | group_settings_ids | realm_setting_group_ids
+    )
```

Comment:
sets are iterable, so I'm not sure it's actually useful to convert to a list here?

Reply:
Yes. Fixed this.